src/base/http/requestparser.cpp

```diff
@@ -350,7 +350,7 @@ bool RequestParser::parseFormData(const QByteArrayView data)
 
     if (headersMap.contains(filename))
     {
-        m_request.files.append({headersMap[filename], headersMap[HEADER_CONTENT_TYPE], payload.toByteArray()});
+        m_request.files.append({headersMap[filename], headersMap[HEADER_CONTENT_TYPE], fromView(payload)});
```

Comment:
Here it is completely incorrect since it will refer to the data after free.

Reply:
Reverted.